README.md

```diff
@@ -13,6 +13,7 @@ TypeScript and JavaScript SDK for the [ChannelApe REST API](https://docs.channel
 - [Variants](#variants)
 - [Businesses](#businesses)
 - [Subscriptions](#subscriptions)
+- [Activities](#activities) (Log arbitrary information on various ChannelApe objects)
```

Comment:
Please change this to the following to better match the channelape docs (https://docs.channelape.io/#bbc83560-59b9-4f6f-8577-0b9b8e78a90a):

```
- [Order Activities](#order-activities)
```